plugins/chat/spec/services/chat/create_message_spec.rb

```diff
@@ -34,21 +34,26 @@
     fab!(:channel) { Fabricate(:chat_channel, threading_enabled: true) }
     fab!(:thread) { Fabricate(:chat_thread, channel: channel) }
     fab!(:upload) { Fabricate(:upload, user: user) }
+    fab!(:user_upload) { Fabricate(:user_upload, upload:, user:) }
     fab!(:draft) { Fabricate(:chat_draft, user: user, chat_channel: channel) }
 
+    fab!(:another_upload) { Fabricate(:upload, user: other_user) }
+    fab!(:another_user_upload) { Fabricate(:user_upload, upload: another_upload, user:) }
+
     let(:guardian) { user.guardian }
     let(:content) { "A new message @#{other_user.username_lower}" }
     let(:context_topic_id) { nil }
     let(:context_post_ids) { nil }
+    let(:upload_ids) { [upload.id] }
     let(:blocks) { nil }
     let(:params) do
       {
         chat_channel_id: channel.id,
         message: content,
-        upload_ids: [upload.id],
-        context_topic_id: context_topic_id,
-        context_post_ids: context_post_ids,
-        blocks: blocks,
+        upload_ids:,
+        context_topic_id:,
+        context_post_ids:,
+        blocks:,
```

Comment:
I see that the rest of the test seems not to use this shorthand notation (i.e., it uses `user: user` instead of just `user:`). I'm not sure you have a convention over this - I can remove the use of shorthands if preferred.

Reply:
this convention is way cleaner, I like, I think it is just that people did not know about this in the past